django/db/models/fields/utils.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
I have this edit and some others ready to push, @czpython can you enable [allow edits form maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on this PR?